webpack.rules.ts

```diff
@@ -28,4 +28,20 @@ export const rules: Required< ModuleOptions >[ 'rules' ] = [
 			},
 		},
 	},
+	{
+		test: /\.riv$/,
+		use: [
+			{
+				loader: 'url-loader',
+			},
+		],
+	},
+	{
+		test: /\.wasm$/,
+		use: [
+			{
+				loader: 'url-loader',
+			},
+		],
+	},
```

Comment:
I presume this is only needed in the renderer as it's meant to be used for rendering Rive animations. I'd like to propose to move the rules to `webpack.renderer.config.ts`:

```patch
diff --git forkSrcPrefix/webpack.rules.ts forkDstPrefix/webpack.rules.ts
index fdb41d46173b4c9617474c9dc3edeb59ea57605a..a3d0b8760b0b2b65dc608d8e4d1b744be32d54d5 100644
--- forkSrcPrefix/webpack.rules.ts
+++ forkDstPrefix/webpack.rules.ts
@@ -28,20 +28,4 @@ export const rules: Required< ModuleOptions >[ 'rules' ] = [
 			},
 		},
 	},
-	{
-		test: /\.riv$/,
-		use: [
-			{
-				loader: 'url-loader',
-			},
-		],
-	},
-	{
-		test: /\.wasm$/,
-		use: [
-			{
-				loader: 'url-loader',
-			},
-		],
-	},
 ];
diff --git forkSrcPrefix/webpack.renderer.config.ts forkDstPrefix/webpack.renderer.config.ts
index 99107518b64ee3878e711e044be104aca7386e0c..2af3c36563f06dcbaca6f7deebd9844b4c5027aa 100644
--- forkSrcPrefix/webpack.renderer.config.ts
+++ forkDstPrefix/webpack.renderer.config.ts
@@ -20,6 +20,13 @@ rules.push( {
 	type: 'asset/inline',
 } );
 
+// Encode Rive and WASM files as base64 data URIs
+// This is mainly used in Rive animations.
+rules.push( {
+	test: /\.(riv|wasm)$/i,
+	type: 'asset/inline',
+} );
+
 export const rendererConfig: Configuration = {
 	devtool: 'source-map',
 	module: {

```

Reply:
I've addressed this in [4c9ae56](https://github.com/Automattic/studio/pull/280/commits/4c9ae568bd6dd6560ec15ae295675e791b4f27d8). 

Webpack isn't able to read `.riv` files to generate data URIs, so I’ve used `asset/resource` instead which will export a URL for the files. Seems like this is the [more modern](https://webpack.js.org/guides/asset-modules/) way to handle assets.